pytorch-qa-chatbot/inference/lib/create_chatbot.py

```diff
@@ -62,7 +91,12 @@ def create_chat_bot(
             host=ts_host, port=ts_port, protocol=ts_protocol, model_name=model_name, verbose=True
         )
     else:
-        tokenizer = LlamaTokenizer.from_pretrained(model_name, use_auth_token=True)
+
+        if "alpaca" in model_name.lower():
```

Comment:
Instead of branching, will see if we can get the tokenizer passed as an argument